packages/spor-input-react/src/ChoiceChip.tsx

```diff
@@ -74,7 +86,12 @@ export const ChoiceChip = forwardRef((props: ChoiceChipProps, ref) => {
         data-hover={dataAttr(state.isHovered)}
         data-focus={dataAttr(state.isFocused)}
       >
-        {icon && <chakra.span __css={styles.icon}>{icon}</chakra.span>}
+        {icon && (
+          <chakra.span __css={styles.icon}>
+            {state.isChecked && checkedIcon ? checkedIcon : icon}
+          </chakra.span>
+        )}
```

Comment:
Om du skal innføre den endringen jeg foreslo over nå, så må du også endre denne koden.

```suggestion
        {icon && (
          <chakra.span __css={styles.icon}>
            {state.isChecked ? icon.checked : icon.default}
          </chakra.span>
        )}
```